src/components/settings/settings.js

```diff
@@ -205,7 +205,7 @@ class SettingsControl extends BaseComponent {
         });
     }
 
-    if (props.isLive && qualityOptions.length === 0) return undefined;
+    if (qualityOptions.length <= 1 && speedOptions.length <= 1) return undefined;
```

Comment:
same Q as above. what is we have 1 quality and many sppeds

Reply:
Yes, we need to do it per option menu - I think it's better to leave it here, per each option and remove the additional condition from the container and item components.

Reply:
@yairans  if we have one quality and many speed - we want to render the button and the menu - the inner item will not be render since i added a check for it